packages/earthdata-login-client/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@cumulus/earthdata-login-client",
+  "version": "1.23.2",
+  "description": "A client for interacting with the Earthata Login service",
+  "private": true,
```

Comment:
This one should be public

Reply:
It wasn't public before, it was part of the API package. If we are going to make it public, then there's work that needs to be done to make sure that we're sure that the API interface really is what we want it to be. Can we create a ticket to come back and make it public later?

Reply:
I really want this public and I would prefer not in a separate ticket, since we won't do that. I think we just need something in the README that says we're not using semantic versioning.

Reply:
Now updated to be public.